tests/flow/test_temporal.py

```
@@ -0,0 +1,550 @@
+        test_cases = [
+            ({'year': 1984},                                   '1984-01-01'),
+            ({'year': 1984, 'month': 10},                      '1984-10-01'),
+            ({'year': 1984, 'week': 10},                       '1984-03-05'),
+            ({'year': 1984, 'month': 10, 'day': 11},           '1984-10-11'),
+            ({'year': 1984, 'week': 10, 'dayOfWeek': 3},       '1984-03-07'),
+            ({'year': 1984, 'quarter': 3, 'dayOfQuarter': 45}, '1984-08-14'),
+            ({'year': 1984, 'quarter': 3 },                    '1984-07-01'),
+            #({'year': 1984, 'ordinalDay': 202},                '1984-07-20'),
+        ]
+
+        for idx, (map_input, expected) in enumerate(test_cases):
```

Ruff (0.11.9)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)